src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -43,20 +43,14 @@ const ParameterField = (props: ParameterFieldProps) => {
 
   if (parameter.multiple || parameter.choices?.length) {
```

Comment:
If parameter.choices is not null, you clearly should display the select because it's not possible to select the options in a textarea

Reply:
Yes, I missed that one. Thx